dune

```diff
@@ -7,4 +7,8 @@
 
 (env
  (static
-  (flags (-ccopt -static))))
+  (flags (-ccopt -static)))
+  (dev
+  (flags (:standard -g)))
+ (release
+  (flags (:standard -O3))))
```

Comment:
Not super high confidence on these

Reply:
The -O3 flag seems to only work for the compilers with Flambda optimizers. See https://github.com/ocaml/dune/issues/1427 . In particular, it crashes on my machine as I do not have Flambda installed. You could change it to `(ocamlopt_flags (:standard -O3))` or we could instruct people to install Flambda. Not even sure if the compiler needs much optimization as it seems to run fast enough as is. Then again, if it doens't make the compilation take much longer, it might be worth it. Have you looked at compile and runtimes at all, when using -O3?

Reply:
No. I just copied this from some tutorial on ocamldebug, haha. I can just remove all the flags from the release build.

Reply:
Sounds good! If we feel like we need some optimization, we can add them back in and throw in a dependency on Flambda.